src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/IQueryStore.cs

```diff
@@ -14,7 +14,7 @@ public interface IQueryStore
 
         Task DeleteAsync<T>(string typeName, string key) where T : QueryProjectionBase;
 
-        Task<long> DeleteManyAsync<T, T1>(string typeName, Expression<Func<T, T1>> property, T1 value) where T : QueryProjectionBase;
+        Task<long> DeleteManyLessThanAsync<T, T1>(string typeName, Expression<Func<T, T1>> property, T1 value) where T : QueryProjectionBase;
```

Comment:
DeleteManyLessThanDateAsync perhaps.

Reply:
`DeleteManyOlderThanGwilliam?`? Tough one to find a simple name for.

Reply:
I think this is OK as you pass in the property you want to compare.